plugins/VstEffect/VstEffectControlDialog.cpp

```diff
@@ -88,18 +88,18 @@ VstEffectControlDialog::VstEffectControlDialog( VstEffectControls * _ctl ) :
 		m_managePluginButton->setCheckable( false );
 		m_managePluginButton->setCursor( Qt::PointingHandCursor );
 		m_managePluginButton->setActiveGraphic( PLUGIN_NAME::getIconPixmap(
-							"track_op_menu" ) );
+							"controls_active" ) );
 		m_managePluginButton->setInactiveGraphic( PLUGIN_NAME::getIconPixmap(
-							"track_op_menu" ) );
+							"controls" ) );
```

Comment:
@karmux You should add two renamed images in `BUILD_PLUGIN()`, in `plugins/VstEffect/CMakeLists.txt`. I think changing `logo.png` to `*.png` will be fine.

Reply:
The problem was only in `master`. Now fixed via 4ca6cdb.